DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -591,6 +594,149 @@ private void PostUIInit()
             GameProcessLogic.GameProcessExited += SharedUILogic_GameProcessExited;
         }
 
+        private void ConnectionManager_ChannelMOTDComplete(object sender, EventArgs e)
+        {
+            RequestChannelList(null);
+            gameChanneListTimer = new Timer(RequestChannelList, null, 30000, 30000);
+        }
+
+        private void RequestChannelList(object state)
+        {
+            string pattern = gameCollection.GetGameListPatternFromIdentifier(localGameID);
+            connectionManager.RequestChannelList(pattern);
+        }
+
+        private void ConnectionManager_ChannelListReceived(object sender, ChannelTopicEventArgs e)
+        {
+            Logger.Log($"GameBroadcastChannel_ChannelListReceived ** {e.ChannelName}, {e.Topic}");
+
+            var cncnetManager = (CnCNetManager)sender;
+
+            // Ensure the topic starts with "GAME " as expected
+            if (!e.Topic.StartsWith("GAME "))
+            {
```

Comment:
In this project, it is preferred not using braces for a single line. Same below.